raincloudy/faucet.py

```diff
@@ -107,8 +106,28 @@ def battery(self):
         return battery.strip('%')
 
     def update(self):
-        """Callback self._controller.update()."""
-        self._controller.update()
+        """Submit GET request to update information."""
```

Comment:
this was formerly `_get_cu_and_fu_status` on the controller. moved it down the `faucets` because each valve needs to call this endpoint for itself.